src/test/FuzzerImpl.cpp

```diff
@@ -644,9 +647,10 @@ TransactionFuzzer::initialize()
                                                          std::vector<int> pks) {
             auto addOfferParameters = [&accountParameters, &offerParameters](
                                           Asset bid, Asset sell, int pk,
-                                          Price price, int64 amount) {
-                offerParameters.emplace_back(
-                    FuzzUtils::OfferParameters(bid, sell, pk, price, amount));
+                                          Price price, int64 amount,
+                                          bool passive = false) {
```

Comment:
Nitpick: don't use a default parameter here. It's overkill for a function that you are going to call a handful of times, all within this one scope.

Reply:
> Nitpick: don't use a default parameter here. It's overkill for a function that you are going to call a handful of times, all within this one scope.

Done.